collectors/googlestackdriver/test/mock.js

```diff
@@ -0,0 +1,31 @@
+process.env.AWS_REGION = 'us-east-1';
+process.env.al_api = 'api.global-services.global.alertlogic.com';
+process.env.ingest_api = 'ingest.global-services.global.alertlogic.com';
+process.env.azollect_api = 'azcollect.global-services.global.alertlogic.com';
+process.env.aims_access_key_id = 'aims-key-id';
+process.env.aims_secret_key = 'aims-secret-key-encrypted';
+process.env.log_group = 'logGroupName';
+process.env.paws_state_queue_arn = "arn:aws:sqs:us-east-1:352283894008:paws-state-queue";
+process.env.paws_extension = 'googlestackdriver';
+process.env.googlestackdriver_endpoint = 'https://test.alertlogic.com/';
+process.env.googlestackdriver_token = 'googlestackdriver-token';
+process.env.collector_id = 'collector-id';
+
+const AIMS_TEST_CREDS = {
+    access_key_id: 'test-access-key-id',
+    secret_key: 'test-secret-key'
+};
+
+const LOG_EVENT = {
+    // Here is where you fill in you mock log event
```

Comment:
No mock log event required here?

Reply:
Like i said in the PR description, this is a POC and does not have tests yet.

Reply:
Oh yep, of course, tests are wip, sorry

Reply:
a sample event will be greatly appreciated. Event without tests =)